linksharing/handler.go

```diff
@@ -35,13 +39,29 @@ type HandlerConfig struct {
 
 	// Templates location with html templates.
 	Templates string
+
+	// TxtRecordTTL is the duration for which an entry in the txtRecordCache is valid.
+	TxtRecordTTL time.Duration
```

Comment:
for the future, but this value is something we could pull out of DNS per TXT record (in your screenshots, you get to choose the TXT record TTL, and we could just use that here, though it would be different per TXT record)

Reply:
We could use the actual txt record ttl if you think that would be a nicer solution.  There would be some small refactoring for the cache.

Reply:
I will make this change in a following PR

Reply:
:+1: